charts/openebs/openebs-pool-disk-loss/engine.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: pool-chaos
+  namespace: default
+spec:
+  # It can be true/false
+  annotationCheck: 'false'
+  # It can be active/stop
+  engineState: 'active'
+  #ex. values: ns1:name=percona,ns2:run=busybox 
+  auxiliaryAppInfo: ''
+  appinfo:
+    appns: 'default'
+    applabel: 'app=busybox'
+    appkind: 'deployment'
+  chaosServiceAccount: pool-disk-loss-sa
+  monitoring: false
+  # It can be delete/retain
+  jobCleanUpPolicy: 'delete'
+  experiments:
+    - name: openebs-pool-disk-loss
+      spec:
+        components:
+          env:
+            - name: ANSIBLE_STDOUT_CALLBACK
+              value: 'default'
+
+            # Enter the App_label, if you set APP_CHECK to true
+            - name: APP_LABEL
+              value: ''
+
+            # Enter the APP_NAMESPACE, if you set APP_CHECK to true
+            - name: APP_NAMESPACE
+              value: ''
+              
+            - name: APP_PVC
+              value: ''
+
+            # This is a chaos namespace which will create all infra chaos resources in that namespace
+            - name: CHAOS_NAMESPACE
```

Comment:
We don't support downward api yet in chaosEngine. cc: @ispeakc0de

Reply:
Can you please removed this downward api?